examples/typescript/store/modules/people.ts

```diff
@@ -29,7 +29,7 @@ export interface Person {
   gender: string
   ip_address: string
   avatar: string
-  addres: PersonAddress
+  address: PersonAddress
```

Comment:
Crazy how this didn't break anything, because this is never actually displayed in the card component or anything.